llm_on_ray/finetune/dpo_funetuing.py

```diff
@@ -0,0 +1,237 @@
+#
+# Copyright 2023 The LLM-on-Ray Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import datasets
+from peft import LoraConfig
+from transformers import AutoModelForCausalLM
+from typing import Dict
+
+IGNORE_INDEX = -100
+
+
+class DPOIntelOrcaProcesser:
```

Comment:
Is this processing class specified for Orca dataset?

Reply:
Yes